From e53948a6909e98b63c63c97749c7d0eb534fa274 Mon Sep 17 00:00:00 2001
From: Michael Orlitzky <michael@orlitzky.com>
Date: Fri, 28 Dec 2018 09:40:36 -0500
Subject: [PATCH 3/4] Ignore "will bind to" lines in addition to "bind to"
 lines.

In amavisd-new-2.11.0-rc1, the informational "bind to..." messages
were changed to say "will bind to..." instead. This commit updates the
@ignore_list_final regular expression to match both forms.
---
 amavis-logwatch | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/amavis-logwatch b/amavis-logwatch
index 6d06793..250a6ba 100644
--- a/amavis-logwatch
+++ b/amavis-logwatch
@@ -2029,7 +2029,7 @@ sub create_ignore_list() {
    push @ignore_list_final, qr/^address modified \(/;
    push @ignore_list_final, qr/^Request: AM\.PDP /;
    push @ignore_list_final, qr/^DSPAM result: /;
-   push @ignore_list_final, qr/^bind to \//;
+   push @ignore_list_final, qr/^(will )?bind to \//;
    push @ignore_list_final, qr/^ZMQ enabled: /;
 
    push @ignore_list_final, qr/^Inserting header field: X-Amavis-Hold: /;
-- 
2.19.2

